pages/instructorQuestionManualGrading/instructorQuestionManualGrading.ejs

```diff
@@ -21,6 +21,11 @@
       <%- include('../partials/questionSyncErrorsAndWarnings'); %>
     </div>
     <div id="content" class="container">
+      <% if (assessment_instance.open) { %>
```

Comment:
Unrelated to this change, but I might consider renaming this page's files to `instructorInstanceQuestionManualGrading` to differentiate between this and `instructorAssessmentQuestionManualGrading`.

Reply:
I'm wondering if it may be a better idea to use ManualGrading as a prefix to help organize these pages? So:
* `instructorManualGradingAssessment`
* `instructorManualGradingAssessmentQuestion`
* `instructorManualGradingInstanceQuestion`
* `instructorManualGradingNextUngradedQuestion`

Thoughts?

Reply:
Either of these sound better than the current naming scheme. I was so confused all the time when I was working on this.

Do we want the file names to correspond somehow with the URLs too? I know @mwest1066 mentioned not being particularly happy with the URLs at some point, so I don't know if that's something we want to revisit.

Reply:
@jonatanschroeder I think that it makes sense to group all the manual grading pages together as you suggest. We could also consider using a deeper hierarchy and having `pages/instructorManualGrading/assessment/assessment.js` and similar. I don't have a strong opinion about this.

Reply:
> @jonatanschroeder I think that it makes sense to group all the manual grading pages together as you suggest. We could also consider using a deeper hierarchy and having `pages/instructorManualGrading/assessment/assessment.js` and similar. I don't have a strong opinion about this.

@mwest1066 Sounds good. Thoughts about changing the URLs as @SethPoulsen mentioned?

Reply:
Do you have a plan for how to change them? So long as we maintain the necessary structure for authz, I'm pretty flexible.